scripts/qa_crowbarsetup.sh

```diff
@@ -2814,27 +2815,29 @@ function custom_configuration()
             if [[ $hacloud = 1 ]] ; then
                 proposal_set_value nova default "['deployment']['nova']['elements']['${role_prefix}-controller']" "['cluster:$clusternameservices']"
 
-
                 # only use remaining nodes as compute nodes, keep cluster nodes dedicated to cluster only
-                local novanodes
-                novanodes=`printf "\"%s\"," $unclustered_nodes`
-                novanodes="[ ${novanodes%,} ]"
+                local novanodes=$unclustered_nodes
 
-                # make sure we do not have SP1 and SP2 compute nodes
+                # make sure we do not pick SP1 nodes on cloud7
                 if [ -n "$deployceph" ] && iscloudver 7 ; then
-                    proposal_set_value nova default "['deployment']['nova']['elements']['${role_prefix}-compute-${libvirt_type}']" "['$sles12plusnode']"
-                else
-                    proposal_set_value nova default "['deployment']['nova']['elements']['${role_prefix}-compute-${libvirt_type}']" "$novanodes"
+                    novanodes=$(comm -1 -2 <(printf "%s\n" ${sles12plusnodes[@]}) <(printf "%s\n" $unclustered_nodes))
                 fi
+
+                if [ -z $novanodes ]; then
+                   complain 105 "No suitable node(s) for ${role_prefix}-compute-${libvirt_type} found."
```

Comment:
-1 travis/bashate reports wrong indention in L2827 L2884 L2999
needs to be 4 spaces